src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -11,6 +11,21 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- For projects that aren't using Microsoft.NET.Sdk, these props files won't have been imported yet.
+       So import them here. -->
+  <ImportGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
+    <Import Project="Microsoft.NET.Sdk.DefaultItems.props" />
+    <Import Project="Microsoft.NET.SupportedTargetFrameworks.props" />
+  </ImportGroup>
+
+  <!-- RestoreProjectStyle is usually set in Microsoft.NET.Sdk.props, but that's not imported for projects that
+       don't use Microsoft.NET.Sdk, since it has a lot of defaults that they may not expect.  So set it here.
+       This does mean that setting a TargetFramework in your project will opt you in to PackageReference restore
+       style. -->
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
```

Comment:
This won't be necessary for C++ if we're disabling nuget.

Reply:
Removed this